tracked/items/Follow Through.json

```diff
@@ -7,18 +7,18 @@
         "uncommon",
         "warframe"
     ],
-    "icon": "icons/en/Follow_Through.c4f54158e0f340645e9f79d974dba7d9.png",
+    "icon": "https://static.wikia.nocookie.net/warframe/images/b/b8/FollowThroughMod.png",
     "thumb": "icons/en/thumbs/Follow_Through.c4f54158e0f340645e9f79d974dba7d9.128x128.png",
     "icon_format": "port",
-    "url_name": "follow_through",
+    "url_name": "Follow_Through",
```

Comment:
do not change `url_name`, otherwise it will change url of the page

Reply:
Fixed!